services/conan/conan-version.service.js

```diff
@@ -0,0 +1,38 @@
+import { renderVersionBadge } from '../version.js'
+import { ConditionalGithubAuthV3Service } from '../github/github-auth-service.js'
+import { fetchRepoContent } from '../github/github-common-fetch.js'
+import { parseLatestVersionFromConfig } from './conan-version-helpers.js'
+
+export default class ConanVersion extends ConditionalGithubAuthV3Service {
+  static category = 'version'
+
+  static route = { base: 'conan/v', pattern: ':packageName' }
+
+  static examples = [
+    {
+      title: 'Conan Center',
+      namedParams: { packageName: 'boost' },
+      staticPreview: this.render({ version: '1.78.0' }),
+      keywords: ['c++'],
+    },
+  ]
+
+  static defaultBadgeData = { label: 'conan' }
+
+  static render({ version }) {
+    return renderVersionBadge({ version })
+  }
```

Comment:
fwiw you could skip the separate function and avoid some calls here by using `renderVersionBadge` directly from the `handle` function as well as for the `staticPreview` key in the example list